dotcom-rendering/src/components/WeatherSlot.tsx

```diff
@@ -19,15 +20,11 @@ interface IconProps {
 const formatTemperature = (value: number, unit: string) =>
 	`${value}°${unit.toLocaleUpperCase()}`;
 
-const formatTime = (dateTime: string, isUS: boolean) =>
-	isUS
-		? new Date(dateTime).toLocaleTimeString('en-US', {
-				hour: 'numeric',
-		  })
```

Comment:
With this change the US edition will now read `1:00 PM` rather than `01:00 PM` or `1 PM` (see the second screenshot above). I've standardised which fields we show and allowed the internationalisation API[^1] to format them according to locale.

Is there a specific requirement for us to drop the minutes from the US edition? If so this should be straightforward to do 👍.

[^1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
Good call. Looking into this further and talking to @HarryFischer we can see that this timestamp wraps onto a new line when using both hours, minutes and the AM/PM markers that the US and AU formatters add.

I've updated to remove the minutes for US and AU, and added a comment explaining why, in [9edcf66](https://github.com/guardian/dotcom-rendering/pull/8395/commits/9edcf66fdf06676b76f2e8271e97e7cdcb1b83ad).

I'm assuming that these timestamps are always "on the hour", i.e. the minutes are always "00", for this to work. If that's not the case please shout!